stackstate_checks_base/stackstate_checks/base/checks/__init__.py

```diff
@@ -21,5 +22,10 @@
     'EventType',
     'HealthStream',
     'HealthStreamUrn',
-    'Health'
+    'Health',
```

Comment:
If I understand correctly, the __all__ is used to make these classes public when importing this module.
Is there a reason on why we would need to make `Health`, `Stateful` and `Transactional` public classes?